index.html

```diff
@@ -16,48 +19,69 @@
 
 <body>
 
- <!-- navigation code here -->
+  <!-- navigation code here -->
 
+  <!-- Navigation seems to be the full width of page while the .home seems to have a gutter of sorts  so use two generic divs as containers -->
+
+  <!-- if you look at Bootstrap 4  they use the class='banner-wrapper' as the class for navigation bars that are fluid so to mimic the professional syntax -->
+  <div class="banner-wrapper">
+    <header class="container navigation">
+      <h1 class='main-header'>SpaceWalkers</h1>
+      <nav>
+        <a href="#" class="">Home</a>
+        <a href="#" class="">About Us</a>
+        <a href="#" class="">Blog</a>
+        <a href="#" class="">Contact</a>
+        <a href="#" class="">Sign in</a>
+      </nav>
+    </header>
+  </div>
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
-
+      <!-- image code here DONT FORGET TO USE ATTRIBUTE ALT="describe img" FOR ACCESSIBILITY -->
+      <img src="img/main-header.png" alt="main header image of earth with a gutter added and image centered">
       <h2>Lift Off</h2>
```

Comment:
in alt tags you will want to describe what the picture is of, not its positioning and styling on the page. a visually impaired user wont know how to reference what a gutter and centered image is in relation to the rest of the page